shipper.py

```diff
@@ -143,15 +142,28 @@ def shipment_new():
-                            data['deposition_id'] = zen_create_depot(env_repository_zenodo_host, env_repository_zenodo_token)
-                            data['deposition_url'] = ''.join((env_repository_zenodo_host.replace('api', 'deposit/'), data['deposition_id']))
-                            zen_add_zip_to_depot(env_repository_zenodo_host, data['deposition_id'], file_name, compendium_files, env_repository_zenodo_token)
-                            # Add metadata that are in compendium in db:
-                            if 'metadata' in current_compendium:
-                                if 'zenodo' in current_compendium['metadata']:
-                                    md = current_compendium['metadata']['zenodo']
-                                    zen_add_metadata(env_repository_zenodo_host, data['deposition_id'], md, env_repository_zenodo_token)
-                            data['status'] = 'deposited'
+                            if data['recipient'] == 'zenodo':
+                                data['deposition_id'] = zen_create_depot(env_repository_zenodo_host, env_repository_zenodo_token)
+                                data['deposition_url'] = ''.join((env_repository_zenodo_host.replace('api', 'deposit/'), data['deposition_id']))
+                                zen_add_zip_to_depot(env_repository_zenodo_host, data['deposition_id'], file_name, compendium_files, env_repository_zenodo_token)
+                                # Add metadata that are in compendium in db:
+                                if 'metadata' in current_compendium:
+                                    if 'zenodo' in current_compendium['metadata']:
+                                        md = current_compendium['metadata']['zenodo']
+                                        zen_add_metadata(env_repository_zenodo_host, data['deposition_id'], md,
+                                                         env_repository_zenodo_token)
+                                data['status'] = 'deposited'
+                            elif data['recipient'] == 'eudat':
+                                data['deposition_id'] = eudat_create_depot(env_repository_eudat_host, env_repository_eudat_token)
+                                data['deposition_url'] = ''.join((env_repository_eudat_host.replace('api', 'records/'), data['deposition_id']))
+                                eudat_add_zip_to_depot(env_repository_eudat_host, data['deposition_id'], file_name, compendium_files, env_repository_eudat_token)
+                                # Add metadata that are in compendium in db:
+                                if 'metadata' in current_compendium:
+                                    if 'eudat' in current_compendium['metadata']:
+                                        md = current_compendium['metadata']['eudat']
+                                        zen_add_metadata(env_repository_eudat_host, data['deposition_id'], md,
```

Comment:
Maybe rename method if it is not only for Zenodo anymore (I assume that is why the name starts with `zen_`).